data/presets/amenity/recycling/container/glass_bottles

```diff
@@ -0,0 +1,35 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{amenity/recycling_container}"
+    ],
+    "moreFields": [
+        "{amenity/recycling_container}"
+    ],
```

Comment:
Interesting, I did not know there was a way to reference values for another preset.
When I was looking into it before, I wondered whether to move some of the default fields to the moreFields section. Why…
- IMO, they are not that useful for this glass bottle container preset (see notes below)
- there is not recommendation on https://wiki.openstreetmap.org/wiki/Altglascontainer
- there is no recommendation on https://wiki.openstreetmap.org/wiki/Tag:amenity%3Drecycling for glass bottle container
- on the combination tab at https://taginfo.openstreetmap.org/keys/recycling%3Aglass_bottles, I cannot see any relevant info that would argue for one for those additional tags being very visible


```suggestion
    "fields": [
        "operator",
        "recycling_accepts"
    ],
    "moreFields": [
        "opening_hours", // moved, AFAIK they nearly always are place along the road / in public paces so they don't have opening hours. Therefore moved to "moreFields".
        "opening_hours/covid19", // moved, same
        "collection_times", // moved, I know of no glass bottle container that has the collection times written on the container/nearby. Its not that relevant, since one does not plan the drop of this way. Therefore moved to "moreFields".
        "colour",
        "covered",
        "indoor",
        "level",
        "manufacturer",
        "material",
        "ref"
    ],
```

For comparison: https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/amenity/recycling_container.json#L3-L9

Reply:
I just copied that section from the existing Green Waste Container preset: green_waste.json
But all your suggestions make sense.